h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -60,11 +94,20 @@ function AnnotationListContent({
     );
   }
 
-  return annotations.map(anno => (
-    <article key={anno.id} className="border rounded p-2">
-      {anno.text}
-    </article>
-  ));
+  return (
+    <>
+      {annotations.map(anno => (
+        <article key={anno.id} className="border rounded p-2">
+          {anno.text}
+        </article>
+      ))}
+      {loading && (
+        <div className="text-center" data-testid="page-loading-indicator">
+          Loading more annotations…
+        </div>
```

Comment:
There are no designs for how the loading indicator for extra pages should look like, so we can probably discuss it with the broader team when we've had the chance to demo it.

Reply:
Using the same indicator as when loading the first page would work fine I think, though I would add some vertical spacing above and below in both cases.